internal/core/src/storage/ChunkCache.cpp

```diff
@@ -66,8 +71,28 @@ ChunkCache::Read(const std::string& filepath,
         auto field_data =
             DownloadAndDecodeRemoteFile(cm_.get(), filepath, false);
 
-        auto chunk = create_chunk(
-            field_meta, field_meta.get_dim(), field_data->GetReader()->reader);
+        std::shared_ptr<Chunk> chunk;
+        auto dim = IsSparseFloatVectorDataType(field_meta.get_data_type())
+                       ? 1
+                       : field_meta.get_dim();
+        if (mmap_enabled) {
+            auto path = std::filesystem::path(CachePath(filepath));
+            auto dir = path.parent_path();
+            std::filesystem::create_directories(dir);
+
+            auto file = File::Open(path.string(), O_CREAT | O_TRUNC | O_RDWR);
```

Comment:
Use mmap manager?

Reply:
> Use mmap manager?

Chunk and ChunkedColumn will be changed a lot if I use mmap here. Will change it to MmapChunkManager later.